bin/zopen-info

```diff
@@ -163,6 +167,7 @@ printPackageInfo() {
     size=$(jq -r '.product.size' "${metadata}")
     commitsha=$(jq -r '.product.commitsha' "${metadata}")
     community_commitsha=$(jq -r '.product.community_commitsha' "${metadata}")
+    package_descriptions=$(jq -r --arg package "$package"  '.descriptions[$package]' "${JSON_DESCRIPTIONS_CACHE}")
```

Comment:
We should probably check if JSON_DESCRIPTIONS_CACHE exists first. You can do that with `if [ -r "${JSON_DESCRIPTIONS_CACHE}" ]